js/marketcaps.js

```diff
@@ -135,4 +119,27 @@ $.get( "https://api.coinmarketcap.com/v1/ticker/?convert=EUR&limit=300", functio
         table.columns.adjust();
         table.responsive.recalc();
     })
-});
\ No newline at end of file
+});
+
+function marketcapsGeneratePriceChangeHtml( priceChange ) {
```

Comment:
remove all these 3 functions, move logic back to table render functions, I think it's much clearer.